.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,82 @@
+name: Lint And Test
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs'
+      - '*.md'
+      - 'website'
+      - '.vscode'
+      - 'scripts'
+      - 'templates'
+      - '.gitignore'
+      - 'performance'
+      - '.github/*.yml'
+      - 'packages/**/*md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/workflows/release.yml'
+
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - master
+    paths-ignore:
+      - 'docs'
+      - '*.md'
+      - '.vscode'
+      - 'website'
+      - 'scripts'
+      - '.gitignore'
+      - 'templates'
+      - 'performance'
+      - '.github/*.yml'
+      - 'packages/**/*md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/workflows/release.yml'
+
+env:
+  NODE_OPTIONS: --max_old_space_size=4096
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:13
```

Comment:
We could run this on multiple versions (9,10,11,12) with a [matrix](https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix).

Reply:
But this will mean much longer builds. Longer builds === productivity loss

Reply:
If we would be something like knex that would make sense but since we use wrapper library we rely on their capabilities there.

Reply:
Okay, we could settle on only having it on release action, we would unlikely hit the quota then